app/build.gradle

```diff
@@ -183,6 +184,10 @@ dependencies {
     debugCompile "com.squareup.leakcanary:leakcanary-android:$leakcanary_version"
     compile "com.squareup.leakcanary:leakcanary-android-no-op:$leakcanary_version"
 
+    // screenshot-tests-for-android : Layout Hierarchy Plugin dependencies
```

Comment:
If this is used for UI Tests, then perhaps we could use androidTestCompile instead of compile command?

Reply:
Seems like http://facebook.github.io/screenshot-tests-for-android/ this page compiles normally. But I have very little idea about gradle, so I'm willing to change it, if need be.

Reply:
it is a convention for libs that are used for UI Tests - it's more for the style of the gradle script.  If it works, it'd be great if we change it.